### ISSUE TITLE
Completion commands default error message

### ISSUE THREAD

**Issue by kdeldycke** [gtalarico/pipenv-pipes#18]
**Describe the bug**
The shell completion code introduced in #10 (and at https://github.com/gtalarico/pipenv-pipes/pull/11/files#diff-3aa7be69f7648738d9c6d40f0b79a305R57) doesn't account for the case in which no environment has been created yet.

In that situation, the code snippet splice the default return message of `pipes` and propose each word as if it was a project.

**To Reproduce**
1. `rm -rf $WORKON_HOME`
2. 
    ```shell
    $ _pipenv-pipes_completions() {
    > COMPREPLY=($(compgen -W "$(pipes --_completion)" -- "${COMP_WORDS[1]}"))
    > }
    $ complete -F _pipenv-pipes_completions pipes
    $ pipes <TAB>
    ~/.virtualenvs  environments             in                       
    No                       found                    pipenv                   
    ```

**Expected behavior**
Instead of:
```shell
$ pipes --_completion
No pipenv environments found in ~/.virtualenvs
```

I guess the `--_completion` option should returns nothing when `$WORKON_HOME` is empty.

**Environment:**
 - OS: macOS 10.13
 - Pipes Version: `pipenv-pipes==0.6.2`

**Comment by gtalarico** [comment]
Fixed on #19  - released on [0.6.3](https://pypi.org/project/pipenv-pipes/0.6.3/)

**Comment by kdeldycke** [comment]
Thanks @gtalarico ! That was fast! 😃
